src/coreclr/jit/gentree.cpp

```diff
@@ -20080,6 +20097,10 @@ GenTree* Compiler::gtNewSimdAbsNode(var_types type, GenTree* op1, CorInfoType si
             assert(compIsaSupportedDebugOnly(InstructionSet_AVX512F));
             intrinsic = NI_AVX512F_Abs;
         }
+        else if (compOpportunisticallyDependsOn(InstructionSet_AVX10v1))
```

Comment:
There's a case missed above in `isEmbeddedMaskingCompatibleHWIntrinsic` where `NI_AVX512F_VL_ConvertToVector128UInt32` is handled and we should also be handling `NI_AVX10v1_ConvertToVector128UInt32`

Reply:
Yes. This seems to have been missed after we merged with main. I have added the handling for the same.